cmd/dumpling/main.go

```diff
@@ -88,6 +89,7 @@ func init() {
 	rootCmd.PersistentFlags().BoolVarP(&noSchemas, "no-schemas", "m", false, "Do not dump table schemas with the data")
 	rootCmd.PersistentFlags().BoolVarP(&noData, "no-data", "d", false, "Do not dump table data")
 	rootCmd.PersistentFlags().StringVar(&csvNullValue, "csv-null-value", "\\N", "The null value used when export to csv")
+	rootCmd.PersistentFlags().StringVarP(&sql, "sql", "s", "", "Dump data with given sql")
```

Comment:
I'm not so sure about reserving `-s` here, since mydumper uses `-s` for `--statement-size` (which we are using `-S` for that (which in mydumper and mysqldump means `--socket`)).

We can't use `-e`, it means `--extended-insert` in mysqldump and `--build-empty-files` in mydumper.

cc @3pointer

Reply:
Let's decide this in a new PR.

Reply:
only leave `--sql` option?

Reply:
good idea...